org.envirocar.app/res/layout/activity_help_layout.xml

```diff
@@ -41,8 +41,8 @@
 
         <TextView
             style="@style/enviroCar.Help.content"
-            android:text="@string/help_content_1"
-            android:autoLink="web"/>
+            android:autoLink="email|web"
```

Comment:
only email works, here. No need of email| web.

Reply:
There is a link too, so we gotta add `` web `` too

Reply:
Yes, we do have a web page with that but for feedback we prefer mail.

Reply:
Do I have to remove the link cause with only `` email `` the link won't work?
sorry I didn't understand properly

Reply:
Works fine. Merged !